apps/spark/src/main/java/com/linkedin/openhouse/jobs/util/AppsOtelEmitter.java

```diff
@@ -0,0 +1,105 @@
+package com.linkedin.openhouse.jobs.util;
+
+import com.linkedin.openhouse.common.OtelEmitter;
+import io.opentelemetry.api.OpenTelemetry;
+import io.opentelemetry.api.common.Attributes;
+import io.opentelemetry.api.metrics.LongCounter;
+import io.opentelemetry.api.metrics.LongGaugeBuilder;
+import io.opentelemetry.api.metrics.LongHistogram;
+import io.opentelemetry.api.metrics.Meter;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.Callable;
+import java.util.concurrent.TimeUnit;
+import lombok.extern.slf4j.Slf4j;
+
+@Slf4j
+public final class AppsOtelEmitter implements OtelEmitter {
```

Comment:
To enable dual emission does this class require some extension or passing List<OpenTelemetry> paras should be sufficient?

Reply:
We will create a different OtelEmitter implementation.